chanzuckerberg/idseq-web
# Add data discovery empty states

# Description

* Add states for first time user and user with no data (projects, samples and visualizations)

# Notes

<img width="2046" alt="Screen Shot 2020-04-07 at 9 34 44 PM" src="https://user-images.githubusercontent.com/37312125/78745039-ed42ad00-7917-11ea-9fee-b8a8edf24e37.png">
<img width="2046" alt="Screen Shot 2020-04-07 at 9 35 03 PM" src="https://user-images.githubusercontent.com/37312125/78745050-f03d9d80-7917-11ea-8a3f-ed9b2c20c0db.png">
<img width="2046" alt="Screen Shot 2020-04-07 at 9 35 11 PM" src="https://user-images.githubusercontent.com/37312125/78745054-f2076100-7917-11ea-99fa-606ad4c16e48.png">
<img width="2046" alt="Screen Shot 2020-04-07 at 9 35 16 PM" src="https://user-images.githubusercontent.com/37312125/78745059-f469bb00-7917-11ea-85dc-76a95b1fe96a.png">


# Tests

* Make sure to start with an empty account and delete key `DiscoveryViewSeenBefore` key from you local storage(in `Developer Tools > Application > LocalStorage`)
* Open `My Data` page and see the modal
* Test:
	* Upload data link
	* Public data link
	* Close button
* Should not see the modal anymore (even on refresh)
* In `My Data`:
	* Navigate to projects tab and see the new no data banner for project an test the link
	* Navigate to samples tab and see the new no data banner for samples, test link
	* Navigate to visualizations tab and see the new no data banner for visualizations, test link (currently redirects to info on saving heatmaps)
